src/ResultView.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+
+const result_category = {
+  fontFamily: "Helvetica",
+  fontSize: 14,
+  color: "#7C7C72",
+};
+
+const func = ({ result }: { result: any }) => {
+  console.log(result.course_of_study ? result.course_of_study.raw : "NA");
+  return (
+    <div>
+      <li className="sui-result">
+        <div className="sui-result__header">
+          <span
+            className="sui-result__title"
+            // Snippeted results contain search term highlights with html and are
+            // 100% safe and santitized, so we dangerously set them here
```

Comment:
Do you remember which part of the documentation said this? Would be good to reference it here.